bands/models.py

```diff
@@ -103,33 +128,30 @@ def uploaded_stage_plot(self):
         return os.path.exists(self.stage_plot_fpath)
 
     @property
-    def w9_extension(self):
-        return extension(self.w9_filename)
-
-    @property
-    def bio_pic_extension(self):
-        return extension(self.bio_pic_filename)
-
-    @property
     def stage_plot_extension(self):
         return extension(self.stage_plot_filename)
 
-    @property
-    def all_badges_claimed(self):
-        return not any(a.is_unassigned for a in self.group.attendees)
 
-    @property
-    def completed_agreement(self):
-        return bool(self.poc_phone)
+class BandPanel(MagModel):
+    band_id = Column(UUID, ForeignKey('band.id'))
+    completed = Column(Boolean, default=False)
+    # This needs to be a nullable integer rather than a nullable boolean to prevent SQLAlchemy from setting a False value
+    # when it's instantiated and saved without this field being set.  An annoying but necessary workaround.
+    wants_panel = Column(Integer, nullable=True, default=None)
```

Comment:
This is no longer a necessary workaround, and we should change this to no longer be ``nullable`` because we don't ever expect this modle to be instantiated without this field being filled in.  Previously this column was part of the ``Band`` model, which did get saved in advance of this field being defined.  So we should change this from a nullable integer to a non-nullable Boolean.